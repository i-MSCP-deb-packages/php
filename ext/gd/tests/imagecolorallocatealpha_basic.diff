001+ string(32) "d3c7021a1c1ac969061381fc86dc4e14"
001- string(32) "b856a0b1a15efe0f79551ebbb5651fe8"