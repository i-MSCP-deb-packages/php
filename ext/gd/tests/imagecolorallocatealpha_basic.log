
---- EXPECTED OUTPUT
string(32) "b856a0b1a15efe0f79551ebbb5651fe8"
int(842163455)
---- ACTUAL OUTPUT
string(32) "d3c7021a1c1ac969061381fc86dc4e14"
int(842163455)
---- FAILED
