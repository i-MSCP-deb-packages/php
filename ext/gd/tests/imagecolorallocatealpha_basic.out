string(32) "d3c7021a1c1ac969061381fc86dc4e14"
int(842163455)